src/components/Tags/__tests__/index.tsx

```diff
@@ -114,8 +116,10 @@ describe('Tags', () => {
       {
         transform: async () => {
           const input = document.getElementById('test')
```

Comment:
We need to drop all document.X and use jest selectors. Our tests mustn't be conditonnals @philibea

Reply:
Solved